README.md

```diff
@@ -398,6 +399,39 @@ To build the documentation, you must install the following products:
 |:------------ |:------------ |
 | doxygen<br/>graphviz<br/>python<br/>xsltproc<br/>md5sum  | [http://www.doxygen.org/](http://www.doxygen.org/)<br/>[http://www.graphviz.org/](http://www.graphviz.org/)<br/>[http://www.python.org/](http://www.python.org/)<br/>[http://xmlsoft.org/XSLT/](http://xmlsoft.org/XSLT/)<br/>[http://www.microbrew.org/tools/md5sha1sum/](http://www.microbrew.org/tools/md5sha1sum/) |
 
+[TOC](#TOC)
+<div id="BuildTagWebpages">
+
+## Building Exiv2 Tag Webpages
+
+Exiv2 provides many built-in metadata tags which are listed in the sub-pages of https://exiv2.org/metadata.html 
+and https://pre-release.exiv2.org/metadata.html. Those tag webpages are generated using tag information 
+extracted from the Exiv2 source code.
+
+The tag webpage build files are in the `<exiv2dir>/doc/templates` directory. If changes are made to 
+tag groups in the Exiv2 source code then the build files need to be updated. Any changes made 
+to individual tags in an existing tag group are automatically included.
+
+To build the tag webpages, first [build Exiv2 from source](#TOC) with the `-DEXIV2_BUILD_SAMPLES=ON` 
+option enabled. This is required as the [taglist](README-SAMPLES.md#taglist) sample program is used when building the tag webpages.
+
+Next, set the `EXIV2HOME` environment variable to the Exiv2 top-level directory.
+For example,
+
+```bash
+$ export EXIV2HOME=<exiv2dir>
+```
+
+Then, change to the `doc/templates` directory and run `make`.
+
+```bash
+$ cd <exiv2dir>/doc/templates
```

Comment:
I tried to follow the steps and I discovered something that could be improved. The Makefile under the templates folder uses the EXIV2HOME variable like this:

```
TAGLIST = ${EXIV2HOME}/build/bin/taglist
```
So it is assuming that the `build` directory has that specific name. I think it would be better to use a variable EXIV2_BINDIR as we do when running the python tests, so that the script can easily find the `taglist` executable. At the moment, even if you follow the instructions step by step, the script does not have a way to know what's your build directory.

Another thing that I noticed is that the Makefile is assuming the existence of some prerequisites: bash, make, xsltproc, python3. It would be nice to mention in this README those depedencies.

Reply:
@piponazo:
Thanks, I have made the updates.